### DISCUSSION TITLE
problem with fmu when using fluent

### DISCUSSION THREAD

**Discussion by Xiuchanhun** [ThummeTo/FMIExport.jl#51] [Q&A]
when I use fluent to simulate by using fmu，there is something wrong with it and show me some message：
From test-0: stepping - Error test_fmi2DoStep: Current communication point ( 5.999999865889549255e-02 s) does not match last evaluation time ( 0.000000000000000000e+00 s) within tolerance  9.999999999999999547e-08. Time rollback is not allowed
and fluent cannot send data to fmu model

**Comment by ThummeTo** [comment]
We need much more information on the problem, like used versions and the full error message.
Further, it is a big help if the FMU is allowed to be shared.
Does the FMU work in other FMI simulators, like e.g. fmPy or OMEdit?
Best regards!

**Comment by ThummeTo** [comment]
(I convert this to discussion, please feel free to open an issue with further information to reproduce the error if this is still an issue)
